.github/workflows/run-integration-tests.yaml

```diff
@@ -0,0 +1,27 @@
+name: run-integration-tests
+
+on:
+  push:
+    branches:
+      - main
+      - variant-data-pipeline
+  workflow_dispatch:
+
+jobs:
+  run-checks:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: data-pipeline
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v4
+
+      - name: Build Docker image 🐳
+        run: |
+          docker build -t data-pipeline .
+
+      - name: run integration tests 🧪
```

Comment:
To see if this would work (specifically, was unsure how the CLI tools would fare), I temporarily added an integration test step to the existing "run-code-checks.yaml".

Results here: https://github.com/reichlab/variant-nowcast-hub/actions/runs/9980645497/job/27582521804